src/components/ErrorText.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import iconHint from "../assets/images/icon-hint.svg";
+
+interface ErrorTextProps {
+  children: string;
+}
+
+const ErrorText: React.FC<ErrorTextProps> = ({ children }) => {
+  return (
+    <span className="form-error-text text-preset-9">
+      <img src={iconHint} alt={iconHint} />
```

Comment:
Do you really want your users to see image source as alt? I think its incorrect. Alt has a special meaning so make sure to add some descriptive value there